operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [FEATURE] Added additional options for experimantal feature KEDA Autoscaling. A global configuration section `kedaAutoscaling`is added to the helm chart. This section can be used to configure the KEDA autoscaling for all components. Nevertheless, the individual configuration for each component is still available and will override the global configuration. #7392
 * [FEATURE] Added experimental feature for deploying [KEDA](https://keda.sh) ScaledObjects as part of the helm chart for the components: distributor, querier, query-frontend and ruler. Autoscaling can be enabled via `distributor.kedaAutoscaling`, `ruler.kedaAutoscaling`, `query_frontend.kedaAutoscaling`, and `querier.kedaAutoscaling`. Requires metamonitoring, for more details on metamonitoring see [Monitor the health of your system](https://grafana.com/docs/helm-charts/mimir-distributed/latest/run-production-environment-with-helm/monitor-system-health/). See [grafana/mimir#7367](https://github.com/grafana/mimir/issues/7367) for a migration procedure. #7282
```

Comment:
since we use the changelog per release, it would be easier for users to understand it if the feature had one changelog entry. Can you update the existing entry and add the PR number at the end of it?